lib/actions/user.js

```diff
@@ -78,13 +79,18 @@ export function fetchAuth0Token (auth0) {
 }
 
 /**
- * Updates the redux state with the provided user data,
- * and also fetches monitored trips if requested, i.e. when
+ * Updates the redux state with the provided user data, including
+ * placing the Home and Work locations at the beginning of the list
+ * of saved places for rendering in several UI components.
+ *
+ * Also, fetches monitored trips if requested, i.e. when
  * - initializing the user state with an existing persisted user, or
  * - POST-ing a user for the first time.
  */
 function setUser (user, fetchTrips) {
   return function (dispatch, getState) {
+    positionHomeAndWorkFirst(user)
```

Comment:
I still don't think the client should be concerned with this, but I won't make this blocking so we can move forward. I do think we should add an issue to otp-middleware that handles this sorting before persisting an update and ensures that there is only one home/work location for any given user.

Reply:
Noted in https://github.com/ibi-group/otp-middleware/issues/136.